charts/signon/templates/deployment.yaml

```diff
@@ -0,0 +1,117 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}
+  labels:
+    app: {{ .Release.Name }}
+spec:
+  replicas: {{ .Values.replicaCount}}
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Release.Name }}
+          image: "{{ .Values.appImage.repository }}:{{ .Values.appImage.tag }}"
+          ports:
+            - name: http
+              containerPort: {{ .Values.appPort }}
+          env:
+            - name: ASSET_HOST
+              value: "{{ .Values.appAssetHost | default (printf "https://www-origin-%s.eks.%s.%s" .Release.Namespace .Values.govukEnvironment .Values.govukDomainExternal) }}"
+            - name: DATABASE_URL
+              valueFrom:
+                secretKeyRef:
+                  name: signon
+                  key: database-url
+            - name: DEVISE_PEPPER
+              value: "secret"
+            - name: DEVISE_SECRET_KEY
+              value: "secret"
+            - name: GOVUK_APP_DOMAIN
+              value: "{{ .Values.appGovukAppDomain | default (printf "%s.eks.%s.%s" .Release.Namespace .Values.govukEnvironment .Values.clusterDomain) }}"
+            - name: GOVUK_APP_DOMAIN_EXTERNAL
+              value: "{{ .Values.appGovukDomainExternal | default (printf "%s.eks.%s.%s" .Release.Namespace .Values.govukEnvironment .Values.govukDomainExternal) }}"
+            - name: GOVUK_APP_NAME
+              value: "{{ .Values.appGovukAppName | default .Release.Name }}"
+            - name: GOVUK_APP_TYPE
+              value: "{{ .Values.appGovukAppType }}"
+            - name: GOVUK_WEBSITE_ROOT
+              value: "{{ .Values.appGovukWebsiteRoot | default (printf "https://www-origin-%s.eks.%s.%s" .Release.Namespace .Values.govukEnvironment .Values.govukDomainExternal) }}"
+            - name: DEFAULT_TTL
+              value: "{{ .Values.appDefaultTtl }}"
+            - name: GOVUK_ASSET_ROOT
+              value: "{{ .Values.appGovukAssetRoot | default (printf "https://assets-%s.eks.%s.%s" .Release.Namespace .Values.govukEnvironment .Values.govukDomainExternal) }}"
+            - name: GOVUK_CONTENT_SCHEMAS_PATH
+              value: "{{ .Values.appGovukContentSchemasPath }}"
+            - name: GOVUK_ENVIRONMENT
+              value: "{{ .Values.govukEnvironment }}"
+            - name: PLEK_SERVICE_CONTENT_STORE_URI
+              value: "{{ .Values.appPlekServiceContentStoreUri | default (printf "http://content-store.%s.%s" .Release.Namespace .Values.clusterDomain) }}"
+            - name: PLEK_SERVICE_PUBLISHING_API_URI
+              value: "{{ .Values.appPlekServicePublishingApiUri | default (printf "http://publishing-api-web.%s.%s" .Release.Namespace .Values.clusterDomain) }}"
```

Comment:
Does Signon actually use this? I couldn't find any reference to /publishing.api/ in the tree.

Reply:
I will double-check, you may be right here